scripts/test_schemas.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+green="$(tput setaf 2)"
+red="$(tput setaf 1)"
+default="$(tput sgr0)"
+checks=4
+
+until [ "$checks" == 0 ]; do
+    response="$(curl -so /dev/null -w '%{http_code}' http://localhost:5001/status)"
+
+    if [ "$response" != "200" ]; then
+        echo "${red}---Error: Schema Validator Not Reachable---"
+        echo "HTTP Status: $response"
+        if [ "$checks"  != 1 ]; then
+            echo -e "Retrying...${default}\\n"
+            sleep 5
+        else
+            echo -e "Exiting...${default}\\n"
+            exit 1
+        fi
+        (( checks-- ))
+    else
+        (( checks=0 ))
+    fi
+
+done
+
+exit=0
+
+if [ $# -eq 0 ] || [ "$1" == "--local" ]; then
+    file_path="./schemas"
```

Comment:
If the translated schemas are built they fail with a lot of "Found mismatching answer value for label" errors. This should be `./schemas/en` until validation allows those mismatches for translated schemas.